Define '--noxattr' param

tpryaminin opened the pull request:
Even if pass option --sigfile, evmctl try to set xattr. option to avoid this has been added in this pull.

mimizohar commented:
The existing code seems to work just fine with '-n'.   With your change "--noxattr" can be used in lieu of the '-n'.  Please update the "usage()" as well.

tpryaminin commented:
> The existing code seems to work just fine with '-n'. With your change "--noxattr" can be used in lieu of the '-n'. Please update the "usage()" as well.

Done.

mimizohar commented:
> > The existing code seems to work just fine with '-n'. With your change "--noxattr" can be used in lieu of the '-n'. Please update the "usage()" as well.
> 
> Done.

Please merge the two patches and add a commit message, example shown below.   Also move the new opts[]  definition to earlier in the list, where letters are used.

Subject: Define '--noxattr' param 

Define '--noxattr' as a synonym for the 'n' getopt_long() value.   Update usage() to reflect the change.

tpryaminin commented:
> > > The existing code seems to work just fine with '-n'. With your change "--noxattr" can be used in lieu of the '-n'. Please update the "usage()" as well.
> > 
> > 
> > Done.
> 
> Please merge the two patches and add a commit message, example shown below. Also move the new opts[] definition to earlier in the list, where letters are used.
> 
> Subject: Define '--noxattr' param
> 
> Define '--noxattr' as a synonym for the 'n' getopt_long() value. Update usage() to reflect the change.

I have made a force push of merged patches

mimizohar commented:
> > > > The existing code seems to work just fine with '-n'. With your change "--noxattr" can be used in lieu of the '-n'. Please update the "usage()" as well.
> > > 
> > > 
> > > Done.
> > 
> > 
> > Please merge the two patches and add a commit message, example shown below. Also move the new opts[] definition to earlier in the list, where letters are used.
> > Subject: Define '--noxattr' param
> > Define '--noxattr' as a synonym for the 'n' getopt_long() value. Update usage() to reflect the change.
> 
> I have made a force push of merged patches

@Prodiks The first line in the patch is the "Subject:"  line, which should be followed by a blank line.  The 3rd line in the patch is the patch description.  The patch should look like:

```
Define '--noxattr' param

Define '--noxattr' as a synonym for the 'n' getopt_long() value. Update usage() to reflect the change.
```

Please fix the patch.  I'll keep this pull request open for another week.

tpryaminin commented:
Excuse me.
I hope I did everything right.
@mimizohar

mimizohar commented:
@Prodiks  Please add your "Signed-off-by" tag, otherwise the patch is fine.

tpryaminin committed:
Define '--noxattr' param

Define '--noxattr' as a synonym for the 'n' getopt_long() value. Update usage() to reflect the change.

Signed-off-by: Timofey Pryaminin <tpryaminin@gmail.com>

tpryaminin commented:
> @Prodiks Please add your "Signed-off-by" tag, otherwise the patch is fine.

@mimizohar Done.

mimizohar commented:
@Prodiks Wrong branch.  Patches should be against next-testing.  Reverted pull request. 

Please send a new pull request against next-testing.
